src/QsCompiler/CommandLineTool/QsCommandLineTool.csproj

```
@@ -8,6 +8,22 @@
     <OutputType>Exe</OutputType>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <Authors>Microsoft</Authors>
+    <Description>Microsoft's Q# compiler tool.</Description>
```

How about 
```suggestion
    <Description>Microsoft's Q# command line compiler.</Description>
```
to avoid confusion?